src/js/stores/preferences.js

```diff
@@ -99,6 +99,10 @@ define(function (require, exports, module) {
             return this._preferences;
         },
 
+        get: function (key, unsetValue) {
```

Comment:
JSDoc needed. Since I wrote this, here it is:
```
* Directly get the value for the given preference key, or an unsetValue if the key does not exist.
* 
* @param {string} key
* @param {?string} unsetValue
* @return {?string} 
```